ce/api/geo.py

```diff
@@ -1,6 +1,101 @@
+import logging
+import collections
+import sys
+from collections import OrderedDict
+from threading import RLock
```

Comment:
Correct, we would not. Currently, we enable the use of threading through Flask and have not explicitly ruled out multiple threads for a production environment.